samples/pom.xml

```diff
@@ -34,6 +34,11 @@
 
   <name>Apache Apex Malhar (incubating) Samples</name>
 
+  <properties>
+    <semver.plugin.skip>true</semver.plugin.skip>
```

Comment:
The common to all modules configuration should be part of plugin management and each module may modify it. It is not clear to me why it is necessary to provide semver plugin configuration settings here as semver plugin does not seem to be enabled for the samples module.